pygitguardian/client.py

```diff
@@ -284,6 +285,7 @@ def multi_content_scan(
         and, optionally, filename.
             example: [{"document":"example content","filename":"intro.py"}]
         :param extra_headers: additional headers to add to the request
+        :param params: request parameters
```

Comment:
This method takes rather "high level" objects: it knows what the API expects. For this reason I think it would be nicer if it took a `ignore_known_secrets` (assuming this is the name of the parameter expected by the API) optional boolean and created the `params` argument itself.

Reply:
done